Dockerfile

```diff
@@ -40,6 +40,9 @@ RUN wget ${BAZEL_DOWNLOAD_URL}/${BAZEL_VERSION}/${BAZEL_INSTALLER} && \
 RUN pip install --upgrade pip setuptools wheel && \
     pip install pipenv
 
+# Install black
+RUN pip install black
```

Comment:
Add a check for the python version and then install `black`?

Reply:
Yeah, that seems better. Will make the changes.